core/templates/dev/head/pages/exploration_player/hint_and_solution_modal.html

```diff
@@ -0,0 +1,12 @@
+<div class="oppia-learner-hint-title"><[isHint ? "Hint" : "Solution"]></div>
```

Comment:
Ah, @kevintab95 -- this is failing in production because all directive files need to end with _directive.html. Please fix?